packages/v4/patternfly-docs/content/design-guidelines/components/dropdown/dropdown.md

```diff
@@ -16,18 +15,18 @@ A basic dropdown consists of a toggle control to open and close a menu of action
 
 2. **Menu:** The menu contains a list of actions or links. Menus should optimally contain seven (7) or less items for easy scanning by the user. For longer menus, items may be grouped. See the [menu component guidelines](/components/menu/design-guidelines) for more information about supported options, including use of icons, grouping, and descriptions. The Dropdown component includes many common built-in menu variants. It’s also possible to attach a custom menu by using the [basic panel variant](/components/dropdown/react#basic-panel).
```

Comment:
The basic panel variant link is dead - need to fix it to direct to this URL instead: /components/panel/#basic